docs/api.md

```diff
@@ -2173,6 +2174,9 @@ Contains a boolean stating whether the response was successful (status in the ra
 #### response.request()
 - returns: <[Request]> A matching [Request] object.
 
+#### response.securityDetails()
+- returns: <[Object]> An object with security details associated with the response. All item are preserved in form as returned by Chrome DevTools protocol.
```

Comment:
> All item are preserved in form as returned by Chrome DevTools protocol.

@trnkatomas there are fields that don't make sense to expose in the pptr api, e.g. `certificateId` and array of `Network.SignedCertificateTimestamp` objects.

What are your usecases for this?  Let's expose only the information you care about.

Reply:
Hi, thanks for the comment, I actually need issuer, validity from and to and the subject of the certificate. Shall I just extract these info? I need the issuer for the information how much to trust the certificate and the time of validity for quite the same reason.

Reply:
>  Shall I just extract these information?

@trnkatomas yes, let's extract it

> I need the issuer for the information how much to trust the certificate and the time of validity for quite the same reason.

Thank you for sharing your usecase.